src/features/rewards/walletWrapper/index.tsx

```diff
@@ -45,12 +45,13 @@ import {
   CloseStrokeIcon,
   SettingsAdvancedIcon,
   UpholdColorIcon,
-  UpholdSystemIcon,
-  LoveIcon,
-  GiftIcon,
-  MegaphoneIcon
+  UpholdSystemIcon
 } from '../../../components/icons'
 
+import giftIcon from './assets/gift'
```

Comment:
how come that we switched from icon to asset?

Reply:
it looks like it's cause the icon is full color now. could possibly move this to a shared full color icon component but i don't think it's causing harm and could be considered feature specific.

Reply:
I would actually like to have these assets go in as separate actual SVG files, for performance reasons. I’ll follow up with how that’ll be possible with our new Webpack config once that’s done in brave-core. We could also get the same gains by using import() syntax. 
Inline SVG should only really be used for small amounts of data when we want to affect change through css.

Reply:
import() syntax will be used in this PR for added svg's once https://github.com/brave/brave-ui/pull/282 has been merged. Thanks!